frontend/src/pages/ApprovedActivityReport/index.js

```diff
@@ -223,9 +223,10 @@ export default function ApprovedActivityReport({ match, user }) {
         : null}
       <Grid row>
         {navigator && navigator.clipboard
-          ? <button type="button" className="usa-button no-print" disabled={modalRef && modalRef.current ? modalRef.current.modalIsOpen : false} onClick={handleCopyUrl}>Copy URL Link</button>
+          ? <button id="approved-URL" type="button" className="usa-button no-print" disabled={modalRef && modalRef.current ? modalRef.current.modalIsOpen : false} onClick={handleCopyUrl}>Copy URL Link</button>
```

Comment:
Not to be picky but should 'URL' be in caps?

Reply:
From JIRA: Use lower case letters and dashes between words. For example, id="example-button"